internal/talos-test/internal/pkg/runner/runner.go

```diff
@@ -0,0 +1,117 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package runner
+
+import (
+	"context"
+	"errors"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+
+	"github.com/talos-systems/talos/pkg/retry"
+)
+
+// ContainerConfigs hold the configs we use to launch our container
+type ContainerConfigs struct {
+	ContainerConfig *container.Config
+	HostConfig      *container.HostConfig
+}
+
+// CommandLocal runs a local binary. Used for osctl cluster create and setup
+func CommandLocal(command string) error {
+	commandSplit := strings.Split(command, " ")
+	log.Println("issuing local command : '" + command + "'")
+
+	_, err := exec.Command(commandSplit[0], commandSplit[1:]...).Output()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// CommandInContainer simply runs a bash command in the hyperkube containers
+// nolint: gocyclo
+func CommandInContainer(ctx context.Context, client *client.Client, runnerConfig *ContainerConfigs, command string) error {
```

Comment:
it's not about this specific PR, but rather a general question.

We do equivalent of `docker run` many times, which has some good overhead. 

Does it make more sense to do `docker run` once with container which sleeps forever, and use `docker exec` to run each command?

Reply:
^ probably a good idea

Reply:
100% something we should do. throwing that into the backlog.